File: Solutions/Easy/717. 1-bit and 2-bit Characters/Solution.cs
public class Solution {
    public bool IsOneBitCharacter(int[] bits) {
        int n = bits.Length;
        if(bits[n - 1] == 1) return false;
        for(int i = 0; i < n; i++){
            if(i == n - 1) return true;
            if(bits[i] == 1){
                i++;
            }
        }
        return false;
    }
}

File: Solutions/Easy/717. 1-bit and 2-bit Characters/Solution.py
class Solution:
    def isOneBitCharacter(self, bits: List[int]) -> bool:
        if(bits[-1] == 1):
            return False
        n = len(bits)
        i = 0
        while(i < n):
            if(i == n - 1):
                return True
            if(bits[i] == 1):
                i += 1
            i += 1
        return False